### DISCUSSION TITLE
No object definition for ruleset

### DISCUSSION THREAD

**Discussion by joenash** [BattlesnakeOfficial/feedback#3] [Bug Reports]
As far as I can tell, there is no object definition for ruleset in the docs. 

`ruleset` is a field of type `object` that appears in a `game` object, and is documented in the game docs with:
```ruleset
object
Information about the ruleset being used to run this game.
Example: {"name": "standard", "version": "v1.2.3"}
```

But has no object definition it self. Is this example exhaustive, i.e. the only fields are name and version?

**Comment by andrewdmaclean** [comment]
Hey @joenash 👋 Just bumping this up to the top of the list and to let you know that we are looking at how to fix these 🐛 bugs

**Comment by chris-bsnake** [comment]
Yup, we need to update the docs

**Comment by aurorawalker** [comment]
Documentation is updated and can be seen here: https://docs.battlesnake.com/references/api#ruleset
